base_vat_sanitized/models/res_partner.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 Akretion (http://www.akretion.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
```

Comment:
IMO (check [https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py#L2](https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py#L2)) you should add the author in the same line as the company.

Reply:
If I do that, the line will be more than 80 characters long, unless I remove some information.
I just wanted to avoid the removal of any information that was already present.

Reply:
Oh, I see. Thanks for clarification.